www/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```

Comment:
This manifest is used for "add to home screen" (a2hs) on Android. If we're gonna have one, we should put a real `name` and `short_name`. I think "IFDB" would work great for both.

Reply:
Sounds like a plan!